src/server/cmd/pachd/main.go

```diff
@@ -300,23 +301,35 @@ func doFullMode(appEnvObj interface{}) error {
 	}
 
 	healthServer := health.NewHealthServer()
-	return grpcutil.Serve(
-		func(s *grpc.Server) {
-			pfsclient.RegisterAPIServer(s, pfsAPIServer)
-			pfsclient.RegisterObjectAPIServer(s, blockAPIServer)
-			ppsclient.RegisterAPIServer(s, ppsAPIServer)
-			cache_pb.RegisterGroupCacheServer(s, cacheServer)
-			authclient.RegisterAPIServer(s, authAPIServer)
-			healthclient.RegisterHealthServer(s, healthServer)
-		},
-		grpcutil.ServeOptions{
-			Version:    version.Version,
-			MaxMsgSize: grpcutil.MaxMsgSize,
-		},
-		grpcutil.ServeEnv{
-			GRPCPort: appEnv.Port,
-		},
-	)
+
+	httpServer, err := pfs_server.NewHTTPServer(address, []string{etcdAddress}, appEnv.PFSEtcdPrefix, blockCacheBytes)
```

Comment:
At the top of this function there's a goro that runs ListenAndServe so that the tracing tools will work, I think we don't need those now that we've got this server going. Would you mind confirm that's the case and if so removing it?